securedrop_client/gui/widgets.py

```diff
@@ -536,3 +572,111 @@ def add_reply(self, reply, files=None):
         Add a reply from a journalist.
         """
         self.conversation_layout.addWidget(ReplyWidget(reply))
+
+
+class ExportSourceMessagesAndFilesAction(QAction):
```

Comment:
Since this does nothing yet, I think it should be removed from the PR. If we want this feature, we should open a ticket and discuss.